src/ORM/Query.php

```diff
@@ -1299,6 +1299,38 @@ public function insert(array $columns, array $types = [])
         return parent::insert($columns, $types);
     }
 
+    /**
+     * Insert multiple rows of data in one call make use of bulk inserts.
+     *
+     * @param  array  $columns The columns to insert as ['columnName', 'columnName']
```

Comment:
Are those `string[]` ?

Reply:
It's an array of string, yes. It's like the other column array for insert.